src/main/java/org/jfrog/hudson/ArtifactoryBuilder.java

```diff
@@ -72,7 +73,14 @@ public DescriptorImpl() {
         @SuppressWarnings("unused")
         @RequirePOST
         public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Item project) {
-            return PluginsUtils.fillPluginCredentials(project, ACL.SYSTEM);
+            try {
+                if (User.get(Jenkins.getAuthentication()).hasPermission(Jenkins.ADMINISTER)) {
```

Comment:
Question -
did you try using the following permission check instead?
```
if (project.hasPermission(Jenkins.ADMINISTER)) {
```
This may also make the null pointer check unnecessary,

Reply:
We cannot use the project since the project is null when coming from Manage Jenkins.